release-notes/opensearch-ml-common.release-notes-2.13.0.0.md

```diff
@@ -0,0 +1,54 @@
+## Version 2.13.0.0 Release Notes
+
+Compatible with OpenSearch 2.13.0
+
+
+### Experimental Features
```

Comment:
These should not be experimental features

Reply:
OK